src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -1,74 +1,81 @@
 :host {
-  @extend %component-host;
-  @apply items-stretch
+	@extend %component-host;
+	@apply items-stretch
     flex
     text-color-1
     shadow-border-bottom
     m-0
     mb-px;
-  transition: background-color $transition;
-  animation: calcite-fade-in $transition;
+	transition: background-color $transition;
+	animation: calcite-fade-in $transition;
 }
 
-:host(:hover) {
-  @apply bg-foreground-2;
+.label {
+	@apply bg-transparent flex flex-auto items-center cursor-pointer focus-base;
+	&:focus {
+		@apply focus-inset;
+	}
+	&:hover {
+		@apply bg-foreground-2;
+	}
 }
 
 .icon {
-  @apply items-center flex my-0 mx-2 opacity-0 cursor-pointer;
-  color: theme("colors.brand");
-  flex: 0 0 auto;
-  line-height: 0;
+	@apply items-center flex my-0 p-1 cursor-pointer;
+	color: theme("colors.brand");
+	flex: 0 0 auto;
+	line-height: 0;
+	&:hover {
+		@apply bg-foreground-2;
+	}
 }
 
 .icon-dot {
-  @apply w-2 m-2;
-  &:before {
-    content: "\2022";
-  }
+	@apply items-center flex w-6 p-2;
+	&:before {
+		@apply opacity-0;
+		content: "\2022";
+	}
 }
 
-:host([selected]) .icon {
-  transition: opacity $transition;
-  @apply opacity-100;
+.icon calcite-icon {
+	@apply opacity-0;
 }
 
-.label {
-  @apply bg-transparent flex flex-auto px-3 py-2 items-center cursor-pointer focus-base;
-  &:focus {
-    @apply focus-inset;
-  }
+:host([selected]) .icon-dot:before,
+:host([selected]) .icon calcite-icon {
+	transition: opacity $transition;
```

Comment:
Are we able to use values from the Tailwind config here?

Reply:
Handling in a separate PR.